pyproject.toml

```diff
@@ -16,6 +16,9 @@ python = "^3.7"
 requests = "^2.26.0"
 PyJWT = { extras = ["crypto"], version = "^2.1.0" }
 loguru = "^0.5.3"
+pydantic = "^1.8.2"
+fastapi = "^0.70"
+fastapi-utils = "^0.2.1"
```

Comment:
Regarding the dependencies, I think it would best that we freeze the versions (removing `^`) so that there will be no unexpected breakdowns for users in the case of a package update

Reply:
`poetry` is smart enough to not allow a breaking package upgrade (an increment to the first non-zero number)
I would rather keep the dependencies dynamic like this to benefit from security and bug fixing updates